node/src/BaseClient.ts

```diff
@@ -250,12 +250,13 @@ import {
 /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
 type PromiseFunction = (value?: any) => void;
 type ErrorFunction = (error: RedisError) => void;
+/* eslint @typescript-eslint/consistent-indexed-object-style: off,  @typescript-eslint/consistent-type-definitions: off */
 export type ReturnTypeRecord = { [key: string]: ReturnType };
 export type ReturnTypeMap = Map<string, ReturnType>;
-export type ReturnTypeAttribute = {
+export interface ReturnTypeAttribute {
```

Comment:
why you changed to interface? did you changed it to all types?

Reply:
Interface is recommended by ts since it's behave like the JS under the hood and led to better performance, unless there's a specific need the recommendation is to use interface.